### PR TITLE
chore/github-templates

### LINKED ISSUE #58: Add GitHub issue/PR/contributing templates
See: https://github.com/Esri/angular-esri-map/tree/master/.github

### PR DESCRIPTION
**Pull request by jwasilgeo**
resolves #58

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+- Describe the proposed changes:
+
+  - Is there an example or test page to demonstrate any new or changed features?
+  - Does your PR include appropriate tests for source code alterations?
```
Comment (tomwayson) [inline .github/PULL_REQUEST_TEMPLATE.md]: - If you're adding or changing a public API, did you update the docs Usage sections of the README? If not, please provide a code snippet that demonstrates how to consume the new or updated API(s).

Comment (jwasilgeo) [reply .github/PULL_REQUEST_TEMPLATE.md]: Perfect.  On its way soon.

Comment (jwasilgeo) [reply .github/PULL_REQUEST_TEMPLATE.md]: Alright, review again please.

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+### Expected behavior
```
Comment (andygup) [inline .github/ISSUE_TEMPLATE.md]: I suggest adding an additional bullet that asks them to describe their environment and version numbers (e.g. React, Angular, Vue, etc). As we know this repo will be used by a bunch of different frameworks.

Comment (jwasilgeo) [reply .github/ISSUE_TEMPLATE.md]: I'm down with that.  Coming soon.

**Comment by jwasilgeo** [comment]
Take another look please.

File: .github/ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+- Describe what you expected or wanted to happen.
+- What you are trying to achieve?
+- Describe your environment/framework and be specific with version numbers (e.g. React v16, Angular v5.0, Vue v2.5, Ember v2, etc.).
```
Comment (andygup) [inline .github/ISSUE_TEMPLATE.md]: I like it! 👍

**Comment by tomwayson** [comment]
The failing build seems to be related to https://github.com/travis-ci/travis-ci/issues/8836#issuecomment-356696146

**Pull request merged**